client/src/styles/renku_navy_theme.scss

```
@@ -0,0 +1,27 @@
+@import "~bootstrap/scss/functions";
+
+[data-bs-theme="navy"] {
```

We can use the `color-mode` mixin from bootstrap here:

```suggestion
@import "~bootstrap/scss/functions";
@import "~bootstrap/scss/mixins";

@include color-mode(navy) {
```